test/object-store/benchmarks/object.cpp

```diff
@@ -154,7 +154,8 @@ TEST_CASE("Benchmark object", "[benchmark][object]") {
     using AnyDict = std::map<std::string, std::any>;
     _impl::RealmCoordinator::assert_no_open_realms();
 
-    InMemoryTestFile config;
+    TestFile config;
+    config.in_memory = true;
```

Comment:
The "new" in-memory lockfile supports a maximum of 32 live versions and this test needs >200.